recipes-graphics/imx-gpu-viv/imx-gpu-viv-6.inc

```diff
@@ -328,7 +328,6 @@ do_install:append:libc-musl() {
 ALLOW_EMPTY:${PN} = "1"
 
 FILES:libclc-imx = "${libdir}/libCLC${SOLIBS}"
-FILES:libclc-imx-dev = "${includedir}/CL/cl_viv_vx_ext.h"
```

Comment:
This is fine but we need a: `RPROVIDER:libopencl-imx-dev = "libclc-imx-dev"`

Reply:
Can you explain why? For backwards compatibility? I think it was wrong to put the header in libclc-imx to begin with. The recipe may have putting all the OpenCL headers in libclc-imx-dev as well (in fact it was trying to put the CL headers in other packages as well), which I guess was pretty wrong and we fixed without adding RPROVIDER.